Sources/ComposableArchitecture/Effects/Throttle.swift

```diff
@@ -23,12 +23,13 @@ extension Effect {
   ///     `false`, the publisher emits the first element received during the interval.
   /// - Returns: An effect that emits either the most-recent or first element received during the
   ///   specified interval.
-  public func throttle<S: Scheduler>(
+  public func throttle<S: Scheduler & Sendable>(
```

Comment:
The scheduler also travels across concurrency boundaries, so it needs to be sendable. Dispatch queues _are_ sendable, but run loops are not. Combine's `ImmediateScheduler` is also not sendable, but our `.immediate` scheduler is, as is our `TestScheduler`, so I think this is safe to do, it'll just mean folks shouldn't throttle with a run loop.